utils/build.ps1

```diff
@@ -1606,6 +1608,52 @@ function Build-LLVM([Platform]$Platform, $Arch) {
     }
 }
 
+function Build-Sanitizers([Platform]$Platform, $Arch) {
+  $BareTarget = $Arch.LLVMTarget.Replace("$AndroidAPILevel", "")
```

Comment:
I believe that @andrurogerz is working on creating a `ModuleTriple` parameter in the architecture definition to help with this.

Reply:
Oh good, yeah this difference is nifty. The unified build used to pass it via `LLVM_RUNTIMES_TARGET`
[here](https://github.com/llvm/llvm-project/blob/9bb3c62948d0c6dd3fb233186ae6a79744012b8f/llvm/runtimes/CMakeLists.txt#L409), but in the standalone build CMake reports it as unused. Will drop it here.

Reply:
Done